packages/syft/src/syft/core/node/common/client_manager/dataset_api.py

```diff
@@ -459,15 +475,16 @@ def delete(self, name: str) -> bool:
         if asset_id is None:
             raise KeyError(f"The asset with name `{name}` does not exists.")
 
```

Comment:
Q:  L470 , does we delete all the assets in a datasets, I think we are fetching only the first asset_id , as dataset might have many assets?